app/routes/users-index.vue

```diff
@@ -0,0 +1,45 @@
+<template lang="html">
+  <div class="users-index">
+    <div class="section">
+      <div class="users-container">
+        <div class="users-header">
+          <h1 class="users-header__text">Users</h1>
+        </div>
+        <div class="grid">
+          <div v-for="user in users" class="grid__item">
```

Comment:
Because of the way that `redux-thunk-rest` stores items, this should loop through `users.items`